discovery/README.md

```diff
@@ -2,50 +2,18 @@
 
 The Discovery agent finds deployed API Proxies in Apigee then sends them to API Central
 
-## Build and run
+## Executing
 
-The following make targets are available
-
-| Target          | Description                                                    | Output(s)                     |
-| --------------- | -------------------------------------------------------------- | ----------------------------- |
-| dep             | downloads all dependencies needed to build the discovery agent | /vendor                       |
-| test            | runs go test against all test files int he repo                | test results                  |
-| update-sdk      | pulls the latest changes to main on the SDK repo               |                               |
-| build           | builds the binary discovery agent                              | bin/apigee_discovery_agent    |
-| apigee-generate | generates the models for the Apigee APIs                       | pkg/apigee/models             |
-| docker-build    | builds the discovery agent in a docker container               | apigee-discovery-agent:latest |
-
-### Build (Docker)
+The Apigee Discovery Agent is built and distributed as a docker image. To execute the agent with the image the following command may be executed, updating the version.
 
 ```shell
-make docker-build
+docker run --env-file env_vars -v `pwd`/keys:/keys ghcr.io/axway/apigee_discovery_agent:v0.1.31
 ```
 
-### Run (Docker)
+If using a local specs path mount into the /specs volume and set the `APIGEE_SPECCONFIG_LOCALPATH` variable to `/specs`.
```

Comment:
this  one too maybe?
"If using a local specs path, mount to the /specs volume and set...